spark/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala

```diff
@@ -499,8 +499,7 @@ class StatsCollectionSuite
     ("BINARY", "BinaryType"),
     ("BOOLEAN", "BooleanType"),
     ("ARRAY<TINYINT>", "ArrayType(ByteType,true)"),
```

Comment:
Let's add a nested struct test (like double struct or something)

Reply:
This is just testing invalid types, not sure what I would add with structs here

Reply:
STRUCT<STRUCT<a, b>, c> for example?

Reply:
That's a valid type, there's nothing to do with that in this test

Reply:
@Kimahriman Right, sorry for the confusion